### DISCUSSION TITLE
annotation file for image with no object of interest

### DISCUSSION THREAD

**Discussion by dlin95123** [AlexeyAB/darknet#8777] [General]
I have an  object detection project of two classes (person, animal), and I want to use darknet/yolov4 for it. The images can contain person(s) and/or animal(s), or none of these classes. For example, an image may just contain cars and I don't need to detect these cars. 

My question is how do I handle the annotation file for this image that contains no object of interest? Do I create an annotation file for it and its content is empty?

**Comment by stephanecharette** [comment]
See https://www.ccoderun.ca/programming/darknet_faq/#negative_samples and https://www.ccoderun.ca/programming/darknet_faq/#image_markup

**Reply by dlin95123** [reply]
Thank you!
